src/components/Comments/Comment.js

```diff
@@ -46,6 +55,15 @@ const Comment = ({
     level,
     updateVote
   });
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of a loading state is a good practice for handling asynchronous operations. It seems like you're using this state to manage the loading status of direct messages (both regular and anonymous). However, make sure to handle potential errors during these operations and provide appropriate feedback to the user.
```diff
+ const [loading, setLoading] = React.useState({
+   loadingDm: false,
+   loadingDmAnon: false
+ });
+ 
+ const [profile] = React.useContext(Context).profile;
+ const {createSignChat} = useCreateChat();
+ const {sendMessageDM} = useDMMessage();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->